stacks-common/src/types/mod.rs

```diff
@@ -245,11 +260,11 @@ impl CoinbaseInterval {
         }
 
         let mut ht = intervals[0].effective_start_height;
-        for i in 1..intervals.len() {
-            if intervals[i].effective_start_height < ht {
+        for interval in intervals {
```

Comment:
Looks like this is including the 0th element while the original did not. Seems harmless, but how about this instead?
```
for interval in &intervals[1..] {
```

Reply:
Good catch!